README.md

```diff
@@ -1 +1,5 @@
-# Wara-Conference
\ No newline at end of file
+# Wara-Conference
+
+## Live Demo
+
+[Live Demo Link](https://mre55.github.io/Wara-Conference/)
```

Comment:
The Creative Commons license of the design requires that you give appropriate credit to the author. 
Please do this in the README of your project. 😉

Reply:
Kindly, I was trying to follow the challenge breakdown to finish step by step. I thought creating README and recording video about the project is for day 3. Since we only work and have 1 day (6.5 hr) it is hard to finish everything in one day. Can you kindly check the challenge breakdown and my current progress? Here I attach the image. Thank you in advance!
![image](https://user-images.githubusercontent.com/34578631/141974146-cd33155c-f823-4c04-8666-326183756e76.png)

Reply:
@Mre55, 
Capstone projects are to be submitted after you have completed all the project requirements, not each milestone. 🙂 
Please implement all projects requirements, record a Loom video and resubmit for a review.

The challenge breakdown is just to create a daily plan for you to work with so you can finish early 👍

Reply:
@bimbolabuari 

Thank you for your response. I was not aware about that. Sorry I bother you. Thank you again!